hack/dev-env/print-argo-admin-password.sh

```
@@ -2,4 +2,7 @@
 
 set -eo pipefail
 
-kubectl --context vcluster-control-plane -n argocd get secret argocd-initial-admin-secret -o jsonpath='{.data.password}' | base64 -d && echo
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+source ${SCRIPTPATH}/namespaces.sh
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'source' in place of '.' is undefined.

(SC3046)